migrationmanagers/migrationmanagers.go

```diff
@@ -101,18 +101,13 @@ func (m *MigrationManager) ApplyToSchema(migrationRepo migrations.Repository, id
 	return nil
 }
 
-// Apply idempotently applies a migration to the TestTrack server and in-memory
-// schema without recording the version to TestTrack server
-func (m *MigrationManager) Apply(migrationRepo migrations.Repository) error {
+// Sync applies the contents of a migration to the TestTrack server
+func (m *MigrationManager) Sync() error {
```

Comment:
Apply (which applied to the server and in-memory schema representation) was replaced with Sync, which only writes a migration to the server and leaves the in-memory schema alone. We only need to apply to schema when creating a migration (or simulating in the check case, if we get around to that).